lib/zookeeper.js

```diff
@@ -496,6 +515,11 @@ class ZooKeeper extends EventEmitter {
         return NativeZk.ZOO_LOG_LEVEL_DEBUG;
     }
 
+    /** @deprecated @returns {number} 0 */
```

Comment:
I think it is a good idea to leave the deprecated parts of the source code left as-is, no need to add new things there. The updates in `constants.js` are where we want to direct users to.

Reply:
They are removed :+1: